guides/common/modules/proc_increasing-the-logging-level-for-smart-proxy.adoc

```diff
@@ -0,0 +1,20 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="increasing-the-logging-level-for-{smart-proxy-context}"]
+= Increasing the logging level for {SmartProxy}
+
+You can find the log in `/var/log/foreman-proxy/proxy.log`.
```

Comment:
Unless you changed logging to journald: https://docs.theforeman.org/nightly/Administering_Project/index-katello.html#configuring-logging-type-and-layout_admin